### ISSUE TITLE
Dragon doesn't drop Villagers' Eggs and doesn't respawn

### ISSUE THREAD

**Issue by mrbrwnmail** [MLG-Fortress/ExtraHardMode#255]
Minecraft 1.16.4 last version of plugin

`            if (glydiaDropsEggs)
            {
                ItemStack itemStack = new ItemStack(Material.INFESTED_STONE, 2, (short) 120);
                world.dropItemNaturally(entity.getLocation().add(10, 0, 0), itemStack);
            }`

Why does it drop INFESTED_STONE instead of Villagers' eggs?
And it doesn't respawn anyway. 

Config:
`  EnderDragon:
    Respawns: true
    Drops Dragonegg: true
    Drops 2 Villager Eggs: true
    # Dragon summons minions including blazes and zombies. Can also aggro nearby endermen!
    Harder Battle: false
    # More diverse mobs are summoned (blazes, zombies, and skeletons). Harder Battle must be set to true, or this option has no effect.
    Alternative Minions: false
    # Announces in chat when someone is challenging the dragon or has beaten her.
    Battle Announcements: true
    # Block building in the end to prevent players from building big protective structures.
    No Building Allowed: true
    # Sets the health of the Ender Dragon
    Health: 100`

**Comment by RoboMWM** [comment]
what I'm confused, what are villager eggs and infested stone? I don't recall any PRs modifying the ender dragon's drops... hmm.

**Comment by RoboMWM** [comment]
ok, guess these things have existed since the inception of this plugin.

Not sure what villager eggs are. I think it must've been MONSTER_EGG or something like that which is what silverfish would spawn out of - which the 1.13 rename is INFESTED_STONE.

**Comment by RoboMWM** [comment]
https://github.com/MLG-Fortress/ExtraHardMode/commit/c368c2a3a231dd9c2172b9d8f862c49a44f4a8a6#diff-61c302b18581597216f915be6a51e223f87b5372421467f9509c381ca89c442bL180

If you can figure out what the original intent there was that'd be appreciated.

**Comment by mrbrwnmail** [comment]
I don’t think so. IMHO, it was like a temporary fix when was migrating to 1.13 version. Now you should set back this one argument and change it to drop villagers egg (it was like award for defeating dragon).

**Comment by RoboMWM** [comment]
> Now you should set back this one argument

??? Not sure what you're saying here

> I don’t think so. 

You're gonna have to elaborate. As you can clearly see in that commit, I changed it from `MONSTER_EGG` to `INFESTED_STONE` which _is_ the 1.13 name of silverfish-infested stone blocks. It certainly _is not_ a temporary fix when migrating to 1.13.

**Comment by RoboMWM** [comment]
This has been here for over 8 years (i.e. since the beginning of time for this plugin). https://github.com/MLG-Fortress/ExtraHardMode/blame/0e6bc904dd115d2547b07549b8912fa616f2a4fa/src/main/java/com/extrahardmode/features/monsters/Glydia.java#L180
